src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -488,9 +490,30 @@ def genFunc(pattern, iterable):
                 if i.startswith(pattern):
                     yield i
 
+        # NOTE: If the 'dirFilterIncl' is non empty then the cleaning process will
+        #       be enclosed only in this part of the tree and will ignore anything
+        #       from /store/unmerged/ which does not belong to the included filter
+        # NOTE: 'dirFilterExcl' is always applied.
+
+        # Merge the additional filters into a final set to be applied:
+        dirFilterIncl = set(self.msConfig['dirFilterIncl'])
+        dirFilterExcl = set(self.msConfig['dirFilterExcl'])
+        dirFilterAll = dirFilterIncl - dirFilterExcl
```

Comment:
This won't work as expected:
```
In [8]: set([]) - set(["malta"])
Out[8]: set()
```

Maybe the best would be to:
* if an include dir is provided: simply match that dir against the RSE dump, forget all the rest (including any possible directory exclusion). Honestly speaking, we will likely NEVER use this option once the service is stable in production.
* else-if an exclude dir is provided, match everything from the RSE dump but the exclude directories.

Note the `else-if` though.

Reply:
Todor, as chatted over slack, this code already takes the exclusion list into account (and higher precedence over the inclusion list). Thanks!